### PR TITLE
Dev: Alert & Button

### PR DESCRIPTION
**Pull request by NandhiniKrishnamoorthy**
Please review my Alert and Button component

Alert

image

Button

image

### CODE REVIEW

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to me!

File: css/button.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
+﻿.button-main{
+    padding: 5px 10px 5px 10px;
+    border-radius: 10px;    
+    margin-left: 1rem;
+    margin-right: 1rem;
```
Comment (Naveenchandar) [inline css/button.css:5]: Please use margin shorthand

File: css/button.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    .secondary-button:hover {
+        background-color: #5200cc;
+        color: white;
+        cursor: pointer;
+    }
+
+.primary-button:hover {
+    background-color: #5200cc;
+    color: white;
```
Comment (Naveenchandar) [inline css/button.css:27]: Please use hex code for consistency

File: css/button.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+.primary-button {
+    background-color: var(--primary-button-color);
+    color: white;
+}
+
+.secondary-button {
+    border: var(--primary-button-color) 1px solid;
+}
+
+    .secondary-button:hover {
```
Comment (Naveenchandar) [inline css/button.css:19]: Remove extra space.

File: css/button.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+.button-link {
+    text-decoration: underline;
+    font-size: 1.25rem;
+    text-underline-position: under;
+    color: #5200cc;
+    cursor: pointer;
+}
+
+    .button-link:hover {
```
Comment (Naveenchandar) [inline css/button.css:39]: Remove extra space.

File: css/button.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+.float-button{
+    margin-left: 1rem;
+    background-color: white;
+    color: blue;
+    border-radius: 50%;  
+    border:none;
+    padding: 0px 0px 0px 0px;
+}
+
+    .float-button:hover {
```
Comment (Naveenchandar) [inline css/button.css:53]: Remove extra space.

**Review by DevNirah7 — APPROVED** [review]
Looks good to commit!

**Pull request merged**